simtel_pipeline.py

```diff
@@ -14,16 +14,24 @@
 import events_image
 import mc_shower
 
+# from digicamviewer.viewer import EventViewer
+
 from optparse import OptionParser
 import os
 
 
+def files(path):
+    for file in os.listdir(path):
+        if os.path.isfile(os.path.join(path, file)):
+            yield file
+
+
 if __name__ == '__main__':
 
     parser = OptionParser()
     parser.add_option("-p", "--path", dest="directory",
                       help="directory to data files",
-                      default='../../../sst-1m_simulace/data_test/ryzen_testprod/0.0deg/Data/')
+                      default='/mnt/data1/jurysek/simulace_cta_sst/sim_telarray/cta-prod3-sst-dc-2018/0.0deg/Data/')
```

Comment:
how can we avoid putting defaults, that point to some persons private paths. So that our software also works, when a new user wants to try it out?